eclipse/site/pom.xml

```diff
@@ -171,7 +171,7 @@
       <plugin>
         <groupId>org.jboss.tools.tycho-plugins</groupId>
         <artifactId>repository-utils</artifactId>
-        <version>${jbossTychoPluginsVersion}</version>
+        <version>0.23.1</version>
```

Comment:
the other Tycho version are 0.26.0, in Fuse Tooling we are using 0.26.0-SNAPSHOT
Why are we still in 0.23.1?